website/docs/r/cloud_azure_integrations.html.markdown

```diff
@@ -253,7 +253,10 @@ Below argument supports the minimum metric polling interval of 3600 seconds
 
 All `integration` blocks support the following common arguments:
 
-* `metrics_polling_interval` - (Optional) The data polling interval in seconds.
+* `metrics_polling_interval` - (Optional) The data polling interval **in seconds**.
+
+-> **NOTE** For more information kindly refer to [microsoft-azure-integrations](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations/) documentaion.
+
```

Comment:
please repeat this
https://github.com/newrelic/terraform-provider-newrelic/pull/2649/files#r1591305181

Reply:
updated the note message